src/solution.c

```diff
@@ -649,7 +690,7 @@ void solution_setup()
 
   SETTING("solution", "soln_freq", soln_freq, TYPE_FLOAT);
   SETTING("solution", "output_every_n_obs", obs_output_divisor, TYPE_INT);
-
+  
```

Comment:
Trailing whitespace (sublime has a plugin to show this, otherwise its easy to leave behind)